libtcmu_config.c

```diff
@@ -373,6 +373,7 @@ int tcmu_load_config(struct tcmu_config *cfg, const char *path)
 	len = tcmu_read_config(fd, buf, TCMU_MAX_CFG_FILE_SIZE);
```

Comment:
We do not need the fd open after this line.

Can you move the close(fd) here, and then delete the one right after the closing } below.